pipelines/pviMetrics.yaml

```diff
@@ -7,10 +7,11 @@ description: |
   file deleted once the Great Calibrate Refactor has completed.
 tasks:
   analyzeCalibrateImageMetadata:
-    class: lsst.analysis.tools.tasks.MetadataAnalysisTask
+    class: lsst.analysis.tools.tasks.TaskMetadataAnalysisTask
```

Comment:
This is my only "mandatory" change, as otherwise `${DRP_PIPE_DIR}/pipelines/HSC/DRP-RC2.yaml` fails....
Can you also update `calexpMetrics.yaml` and `visitQualityCore.yaml`, please?